dashboard/lib/logic/task_grid_filter.dart

```diff
@@ -87,29 +79,6 @@ class TaskGridFilter extends FilterPropertySource {
   RegExp get hashFilter => _hashProperty.regExp;
   set hashFilter(RegExp regExp) => _hashProperty.regExp = regExp;
 
-  /// The [showAndroid] property is a boolean that indicates whether to display tasks produced
-  /// by an Android stage in the devicelab.
-  /// This property will filter out columns on the build dashboard.
-  bool get showAndroid => _androidProperty.value;
-  set showAndroid(bool value) => _androidProperty.value = value;
-
-  /// The [showIos] property is a boolean that indicates whether to display tasks produced
-  /// by an iOS stage in the devicelab.
-  /// This property will filter out columns on the build dashboard.
-  bool get showIos => _iosProperty.value;
-  set showIos(bool value) => _iosProperty.value = value;
-
-  /// The [showWindows] property is a boolean that indicates whether to display tasks produced
-  /// by a Windows stage in the devicelab.
-  /// This property will filter out columns on the build dashboard.
-  bool get showWindows => _windowsProperty.value;
-  set showWindows(bool value) => _windowsProperty.value = value;
-
-  /// The [showCirrus] property is a boolean that indicates whether to display tasks produced
-  /// by the Cirrus stage in the devicelab.
-  /// This property will filter out columns on the build dashboard.
-  bool get showCirrus => _cirrusProperty.value;
-  set showCirrus(bool value) => _cirrusProperty.value = value;
```

Comment:
You'll need to run https://github.com/flutter/cocoon/blob/main/format.sh to format the code

Or you can add to your `.bashrc`, `alias ff=dart format --set-exit-if-changed --line-length=120 .`

Reply:
Thanks for the instruction. Have changed the format.